server/sonar-web/src/main/js/apps/projectActivity/components/GraphsTooltips.js

```diff
@@ -98,7 +98,7 @@ export default class GraphsTooltips extends React.PureComponent {
       <BubblePopup customClass={customClass} position={{ top, left, width: TOOLTIP_WIDTH }}>
         <div className="project-activity-graph-tooltip">
           <div className="project-activity-graph-tooltip-title spacer-bottom">
-            <FormattedDate date={this.props.selectedDate} format="LL" />
+            <DateTimeFormatter date={this.props.selectedDate} />
```

Comment:
Ah I remember now, I decided to use `DateTimeFormatter` here instead because the old `FormattedDate` component was displaying a date with format LL and adding a tooltip with format LLL on top. And having a tooltip on top of a tooltip... So I decided to directly show the full date since we have enough space.